execution_engine/CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ All notable changes to this project will be documented in this file.  The format
 * Lower the default gas costs of opcodes.
   - Set the cost for branching opcodes to 35,000 (`br`, `br_if`, `br_table`).
   - Set the cost for call opcodes to 68,000 (`call`, `call_indirect`).
+* Default value for a round seigniorage rate is lowered to `7/175070816`
```

Comment:
We should probably mention that this _preserves_ the existing seigniorage per unit of time due to rounds being cut in half